core/src/test/scala/org/apache/spark/scheduler/DAGSchedulerSuite.scala

```diff
@@ -4595,6 +4595,183 @@ class DAGSchedulerSuite extends SparkFunSuite with TempLocalSparkContext with Ti
     }
   }
 
+  test("SPARK-40082: recomputation of shuffle map stage with no pending partitions should not hang") {
```

Comment:
This line is exceeding 100 chars, and is probably responsible for the build failure.

Reply:
fixed it